hy/core/shadow.hy

```diff
@@ -138,11 +138,17 @@
 (defn not-in [x y]
   (not-in x y))
 
+(defn get [thing key1 &rest keys]
+  (for* [k (+ (, key1) keys)]
+    (setv thing (get thing k)))
+  thing)
```

Comment:
Ah, I see. We want to assert that get has at least one key. In the case of zero keys it might make sense to just return the thing. But we'd have to make the special form do that too. But the slightly optimized version I wrote above might perform better.